### ISSUE TITLE
False positives when linting HTML with multiple bootstrap candidates (v1.0.0)

### ISSUE THREAD

**Issue by codeworrior** [SAP/ui5-linter#335]
## Expected Behavior

When I lint an HTML page with a UI5 2.0-compliant bootstrap, I don't get findings from the UI5 linter.

## Current Behavior

When I lint for example sap.ui.core, I get two errors in this [unit test page](https://github.com/SAP/openui5/blob/main/src/sap.ui.core/test/sap/ui/core/qunit/bootstrap/ResourceRoot_ResourcesURL_ResOnly.qunit.html#L28-L38), for the two script tags for ui5loader and ui5loader-autoconfig. But the runtime only takes the script with `id="sap-ui-bootstrap"` into account and ignores the others (wrt. configuration).

Shouldn't the linter first determine which script tag will be evaluated by the runtime and only check that one? 
 
## Steps to Reproduce the Issue

Just lint sap.ui.core (master or main).

## Context

- UI5 linter version: **`1.0.0`**

## Log Output / Stack Trace
(The following lines are not directly from the log, but from a report based on the JSON output)
```
37:3: Missing bootstrap parameter 'data-sap-ui-async'
38:3: Missing bootstrap parameter 'data-sap-ui-async'
```

**Comment by RandomByte** [comment]
We decided to always analyze both script tags, expecting that this wouldn't be used (correctly) in the wild anyways. I guess we were wrong. Shouldn't be a problem fixing it 👍

**Comment by codeworrior** [comment]
Cool, thank you!
